lighthouse-core/audits/byte-efficiency/unused-javascript.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const ByteEfficiencyAudit = require('./byte-efficiency-audit');
+
+const IGNORE_THRESHOLD_IN_BYTES = 2048;
```

Comment:
no research it's just inline with the rest of the byte efficiency audits to not be too noisy about small things that have minimal savings (others are also 2kb or 4kb)